solvebio/cli/data.py

```diff
@@ -162,7 +162,8 @@ def _upload_folder(
                 remote_parent = Object.get_by_full_path(
                     remote_folder_full_path, assert_type="folder"
                 )
-                Object.upload_file(local_file_path, remote_parent.path, vault.full_path)
+                Object.upload_file(local_file_path, remote_parent.path,
+                        vault.full_path, archive_folder=archive_folder)
```

Comment:
seems like archive_folder kwarg is missing for this function (_upload_folder)

Reply:
yuup. Fixed!